### PR TITLE
Disable action API when admin is disabled

### PR DESCRIPTION
**Pull request by mythmon**
Fixes bug 1248630.

### CODE REVIEW

File: normandy/recipes/tests/test_api.py [hunk 1]
```diff
@@ -101,9 +101,15 @@ def test_it_cant_edit_actions_in_use(self, api_client, settings):
 
         res = api_client.patch('/api/v1/action/active/', {'implementation': 'foobar'})
         assert res.status_code == 403
+        assert res.data == {
+            'detail': 'Cannot edit this object while it is in use'
```
Comment (Osmose) [inline normandy/recipes/tests/test_api.py]: I've been on the fence about testing specific messages being returned by the API, but it seems like you fall on the "yes, do that" side of that question?

**Comment by Osmose** [comment]
r+, nice work!

File: normandy/recipes/tests/test_api.py (hunk 1 above)
Comment (mythmon) [inline normandy/recipes/tests/test_api.py]: I want to assert that the right kind of error got thrown. I want to make sure this is an "object in use" error, and not an "admin disabled" error. We could also this with something like `assert res.data['detail'] == NotInUse.message`, if changing error strings ends up being annoying.

Comment (Osmose) [inline normandy/recipes/tests/test_api.py]: I like that suggestion and will use it in future PRs when I can. :D

**Pull request merged**